<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registerWidget</class>
 <widget class="QWidget" name="registerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="userNameLabel">
       <property name="text">
        <string>username</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="passwordLabel">
       <property name="text">
        <string>password</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="passwordLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="checkLabel">
       <property name="text">
        <string>check password</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="checkLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>register</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>hide Password</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="resLabel">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
